mayastor/src/nexus_uri.rs

```diff
@@ -89,104 +89,17 @@ impl RpcErrorCode for BdevCreateDestroy {
     }
 }
 
-/// enum type of URL to args we currently support
-#[derive(Debug)]
-pub enum BdevType {
-    /// you should not be using this other than for testing
```

Comment:
Could these comments be moved to StorageUri::parse()? It'd be a shame to lose them.

Reply:
Good Idea. Will do.